testing/trino-testing/src/main/java/io/trino/testing/AbstractTestQueries.java

```diff
@@ -500,32 +500,6 @@ public void testMinByN()
                 "SELECT orderkey FROM orders ORDER BY totalprice ASC LIMIT 2");
     }
 
-    @Test
-    public void testHaving()
-    {
-        assertQuery("SELECT orderstatus, sum(totalprice) FROM orders GROUP BY orderstatus HAVING orderstatus = 'O'");
-    }
-
-    @Test
-    public void testHaving2()
```

Comment:
sweet naming :p

Reply:
this is why i caught this when running other tests :)